sycl/plugins/unified_runtime/ur/adapters/cuda/common.cpp

```diff
@@ -39,7 +39,8 @@ ur_result_t checkErrorUR(CUresult Result, const char *Function, int Line,
     return UR_RESULT_SUCCESS;
   }
 
-  if (std::getenv("SYCL_PI_SUPPRESS_ERROR_MESSAGE") == nullptr) {
+  if (std::getenv("SYCL_PI_SUPPRESS_ERROR_MESSAGE") == nullptr ||
+      std::getenv("UR_SUPPRESS_ERROR_MESSAGE") == nullptr) {
```

Comment:
should this be UR_CUDA_SUPPRESS_ERROR_MESSAGE? if it is UR_ prefixed, should we have it officially documented in the UR spec?

Reply:
Ummm, this is apparently only intended for CI, so I dont think it should be in the spec, or if we need it in UR (do we want to have it in the future UR CI?) Change is good for now, but we might need to think eventually if we want to carry it over to UR CI. 

| `SYCL_PI_SUPPRESS_ERROR_MESSAGE` | Any(\*) | Suppress printing of error message, only used for CI in order not to interrupt errors generated by underlying toolchains;

Reply:
I've left it as UR_ for now, I think we'll be revisiting native error handling in the CUDA adapter soon which seems like a good opportunity to think about this again